counterparty-rs/src/indexer/logging.rs

```diff
@@ -38,7 +38,7 @@ pub fn setup_logging(config: &Config) {
 
         let stderr_layer = layer()
             .event_format(CustomFormatter {
-                timer: ChronoLocal::rfc_3339(),
+                timer: ChronoLocal::new("%Y-%m-%dT%H:%M:%S%.6f%:z".to_string()),
```

Comment:
I actually think millisecond precision everywhere is enough...

Reply:
done